bindings/nodejs/tests/tests/OIC Server.js

```diff
@@ -22,10 +22,7 @@ var testUtils = require( "../assert-to-console" );
 var theResource;
 var anError = {};
 
-console.log( JSON.stringify( { assertionCount: 4 } ) );
-
-testUtils.assert( "strictEqual", soletta.sol_oic_server_init(), 0,
-	"sol_oic_server_init() is successful" );
+console.log( JSON.stringify( { assertionCount: 3 } ) );
```

Comment:
It seems the work on ca5d3a9a0e9d8145bb0b44a540985c9fd2d0ddef missed this. Okay by me.